packages/jupyter/jupyter.2.8.1/opam

```diff
@@ -0,0 +1,59 @@
+opam-version: "2.0"
+maintainer: [
+  "Akinori ABE <aabe.65535@gmail.com>"
+]
+authors: [
+  "Akinori ABE"
+]
+license: "MIT"
+homepage: "https://akabe.github.io/ocaml-jupyter/"
+bug-reports: "https://github.com/akabe/ocaml-jupyter/issues"
+dev-repo: "git+https://github.com/akabe/ocaml-jupyter.git"
+
+build: [
+  [ "dune" "subst" ] {dev}
+  [ "dune" "build" "-p" name "-j" jobs ]
+]
+depends: [
+  "ocaml" {>= "4.10.0" & <= "4.14.1"}
```

Comment:
```suggestion
  "ocaml" {>= "4.10.0" & < "5.0"}
```
We can add a stricter lower bound if it happens to break with new compilers, I would be less restrictive here

Reply:
Thank you. I fixed it.